### ISSUE TITLE
Placeholder images for places that do not have an image

### ISSUE THREAD

**Issue by xavez** [osoc16/LeuvenLeest-frontend#19]
We need to add a placeholder image for places that do not have a photo. Right now, it is hot linked. http://www.leuven.be/binaries/Stadhuis-CoprightLaylaAerts_tcm16-24398.jpg

This has major high priority, assigning @huhu1030 and @elroyK because we cannot just hotlink a copyrighted image. Please fix ASAP :)

**Comment by elroyK** [comment]
There's already a placeholder for places without pictures, those with this exact picture have it as their related image in the DB I think, can @JasperDR confirm ?

**Comment by JasperDR** [comment]
That is correct, this can be changed in the DB by uploading another picture and putting that one in the DB. I'm already working on a function to return the images from backend.

**Comment by xavez** [comment]
Okay, I don’t know the technical details, what needs to happen is that we
- Remove the current hot linked image
- Replace it with a placeholder image we created

As soon as possible because there is a rights issue.

**Comment by elroyK** [comment]
As long as the hot linked image is deleted from the DB, the placeholder will be applied :)

**Comment by JasperDR** [comment]
cleared 'photos' table from DB

**Comment by xavez** [comment]
Great!

However, who thought it was a good idea to put a 1mb picture as a placeholder? This should be a couple of mb’s max.

Not considering the placeholder debacle fixed just yet ;)

https://leuvenleestapp.be/assets/img/placeholder_locations.jpg
